src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {

```

Comment:
```suggestion
public class Lottery {
	private ColorSupplier colorSupplier = new ColorSupplier();
```
тут замість наслідування краще використати композицію